src/tools/pub/pubspec.md

```diff
@@ -395,9 +400,41 @@ the pattern is considered relative to the package's root directory.
 {{site.alert.end}}
 
 {{site.alert.version-note}}
-  Support for the `false_secrets` field was added in Dart 2.15.
+  Dart 2.15 added support for the `false_secrets` field.
 {{site.alert.end}}
 
+### Screenshots
+
+Packages can showcase their widgets or other visual elements using screenshots
+displayed on their package page.
```

Comment:
**nit:** Don't need to say "package page". The context is sufficient.
```suggestion
displayed on their page.
```

Reply:
Done.